pom.xml

```diff
@@ -131,6 +131,16 @@
       <id>repo.jenkins-ci.org</id>
       <url>http://repo.jenkins-ci.org/public/</url>
     </repository>
+    <repository>
+      <id>twitter</id>
+      <url>http://maven.twttr.com</url>
```

Comment:
@michaelneale I think the right thing to do here is make sure this repository is mirrored by the Jenkins project's Artifactory and then we can remove this `<repository>` entry.

I have raised [INFRA-1259](https://issues.jenkins-ci.org/browse/INFRA-1259)